css-forms-1/Overview.bs

```diff
@@ -843,8 +843,14 @@ ISSUE: Refine meter, progress, switch and range input styling.
 
 ```css
 input:is([type=checkbox]:not([switch]), [type=radio]) {
-    width: 1em;
-    height: 1em;
+    /* These size rules ensure accessibility by following WCAG rules:
+    * https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html */
+    block-size: max(1em, 24px);
+    inline-size: max(1em, 24px);
+
+    /* This font-size rule ensures the checkmark fills the containing box. */
+    font-size: max(1em, 24px);
+
```

Comment:
font-size would control both block-size and inline-size technically, so this would have the same effect...

```
    block-size: 1em;
    inline-size: 1em;
    font-size: max(1em, 24px);
```

cc @fantasai to look at this